### ISSUE TITLE
Can't install from pypi and clonming the project.

### ISSUE THREAD

**Issue by varokr** [xtekky/gpt4free#871]
# Installation fails every time here, 

```Building wheels for collected packages: quickjs
  Building wheel for quickjs (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [15 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-cpython-311
      creating build\lib.win-amd64-cpython-311\quickjs
      copying quickjs\__init__.py -> build\lib.win-amd64-cpython-311\quickjs
      running build_ext
      building '_quickjs' extension
      creating build\temp.win-amd64-cpython-311
      creating build\temp.win-amd64-cpython-311\Release
      creating build\temp.win-amd64-cpython-311\Release\upstream-quickjs
      "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\bin\HostX86\
x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -DCONFIG_VERSION=\"2021-03-27\" -DCONFIG_BIGNUM -IE:\gpt4fre
e-ui-desktop\venv\include -IC:\Python311\include -IC:\Python311\Include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.301
33\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\shared" "-IC:\Pro
gram Files (x86)\Windows Kits\10\include\10.0.19041.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\winrt" "-IC:\Program Files (x86)\Windows
 Kits\10\include\10.0.19041.0\cppwinrt" /Tcmodule.c /Fobuild\temp.win-amd64-cpython-311\Release\module.obj -Werror=incompatible-pointer-types
      cl : Command line error D8021 : invalid numeric argument '/Werror=incompatible-pointer-types'
      error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\VC\\Tools\\MSVC\\14.29.30133\\bin\\HostX86\\x64\\cl.exe' failed with ex
it code 2
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for quickjs
  Running setup.py clean for quickjs
Failed to build quickjs
ERROR: Could not build wheels for quickjs, which is required to install pyproject.toml-based projects
```

**Comment by AmayaKuro** [comment]
quickjs currently available for python 3.10 only, so if you want to install, you can try downgrade your python to 3.10 ver.

**Comment by varokr** [comment]
> quickjs currently available for python 3.10 only, so if you want to install, you can try downgrade your python to 3.10 ver.
Thanks
